models/rubicon/rubicon_schema.yml

```diff
@@ -24,9 +24,6 @@ models:
       - &block_date
         name: block_date
         description: "UTC event block date of each DEX trade"
-      - &block_month
```

Comment:
why remove this? we will need downstream

Reply:
added back in `block_month`